lib/prawn/embedded_files.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Prawn
+  module EmbeddedFiles
+    # include PDF::Core::EmbeddedFiles
+    # Link to PR (https://github.com/prawnpdf/pdf-core/pull/47)
+
+    def file(src, options = {})
```

Comment:
Please add documentation for this method.

Reply:
Done